packages/client/src/forms/configuration/reducer.ts

```diff
@@ -40,8 +40,8 @@ export interface IDraft {
 }
 
 export interface IFormDraftData {
-  birth: IDraft
-  death: IDraft
+  birth: IDraft | null
+  death: IDraft | null
```

Comment:
I think we'll need to have discussion with @euanmillar about if we want to have default drafts in the db

Reply:
Yes. I will labeled the **Questions** tag in this PR.

Reply:
We will need default drafts to be available